packages/affiliate/tests/php/test_Affiliate.php

```diff
@@ -1,23 +1,20 @@
 <?php
-/**
- * Tests for Jetpack_Affiliate
- */
+use Automattic\Jetpack\Partners\Affiliate;
 
-use Automattic\Jetpack\JITM\Manager as JITM_Manager;
+use PHPUnit\Framework\TestCase;
 
 // Load required class to get the affiliate code
 require_once JETPACK__PLUGIN_DIR . 'class.jetpack.php';
```

Comment:
`JETPACK__PLUGIN_DIR` won't be defined here, as these tests run standalone.

Reply:
Replaced with Constants package.